processors/filterprocessor/filter_processor_test.go

```diff
@@ -0,0 +1,272 @@
+package filterprocessor
+
+import (
+	"context"
+	"testing"
+
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/algorand/go-algorand/data/basics"
+	"github.com/algorand/go-algorand/data/transactions"
+	"github.com/algorand/indexer/conduit"
+	"github.com/algorand/indexer/data"
+	"github.com/algorand/indexer/plugins"
+	"github.com/algorand/indexer/processors"
+)
+
+// TestFilterProcessor_Init_Multi tests initialization of the filter processor with the "all" and "any" filter types
+func TestFilterProcessor_Init_Multi(t *testing.T) {
+
+	sampleAddr1 := basics.Address{1}
+	sampleAddr2 := basics.Address{2}
+	sampleAddr3 := basics.Address{3}
+
+	sampleCfgStr := `---
+filters:
+  - any: 
+    - tag: SignedTxnWithAD.SignedTxn.AuthAddr
+      expression-type: exact
+      expression: "` + sampleAddr1.String() + `"
+    - tag: SignedTxnWithAD.SignedTxn.Txn.AssetTransferTxnFields.AssetSender
```

Comment:
I think it's important to use the msgpack path here. Have you looked into how that would be implemented?

For example, this would be `txn.asnd`.

Reply:
I can take a look for sure.  I think its just different logic in the reflection package.